src/app.js

```diff
@@ -310,6 +310,11 @@ function buildSmallScreenWidget(toolUrl) {
 }
 
 function addWidget() {
+  // widget already exists
+  if ($('#ajas-search01') || $('#ajas-search02')) {
```

Comment:
the class ajas-search-widget is applied to both the small and big widgets, you could use that instead of checking both of these.